.github/workflows/main.yml

```diff
@@ -183,6 +181,8 @@ jobs:
           - TEST=Config
           - TEST=Fsa
           - TEST=Log
+          - TEST=Pretrain
```

Comment:
Why do the Pretrain tests actually depend on TF?

Reply:
I did not care about the "why" yet, but it does.

This is what I got before changing it:
```python3
  File "/u/rossenbach/src/returnn/returnn/tf/layers/basic.py", line 11668, in get_layer_class
    line: raise Exception("unknown layer class %r" % name)
    locals:
      Exception = <builtin> <class 'Exception'>
      name = <local> 'lstm_opt', len = 8
Exception: unknown layer class 'lstm_opt'
```

Reply:
What is the full backtrace?

Reply:
https://gist.github.com/JackTemaki/57a51b9e52451341876c711b2ee1947f

Reply:
Ok this is in `LayerNetworkDescription`. I think I would just remove those tests which use `LayerNetworkDescription`.

Reply:
So far I don't see a good reason why it really needs TF. If there is no good reason, we should maybe fix things and move this over to the base tests.